File: ui-web/src/lib/monetization/starterPack.ts
// TypeScript content with proper types

interface StarterPack {
  name: string;
  contents: string[];
  individualValue: string;
  bundlePrice: string;
  discount: string;
  oneTimePurchase: boolean;
  expectedConversion: string;
}

const starterPack: StarterPack = {
  name: "Starter Pack",
  contents: [
    "500 coins",
    "1 skin",
    "3x power-ups"
  ],
  individualValue: "$9.97",
  bundlePrice: "$4.99",
  discount: "50%",
  oneTimePurchase: true,
  expectedConversion: "5-8%"
};

export function getStarterPackDetails(): StarterPack {
  return starterPack;
}

File: core/internal/monetization/starter_pack.go
package monetization

import "fmt"

type StarterPack struct {
	Name              string
	Contents          []string
	IndividualValue   string
	BundlePrice       string
	Discount          string
	OneTimePurchase   bool
	ExpectedConversion string
}

func GetStarterPack() StarterPack {
	return StarterPack{
		Name: "Starter Pack",
		Contents: []string{"500 coins", "1 skin", "3x power-ups"},
		IndividualValue: "$9.97",
		BundlePrice: "$4.99",
		Discount: "50%",
		OneTimePurchase: true,
		ExpectedConversion: "5-8%",
	}
}

func DisplayStarterPackDetails(pack StarterPack) {
	fmt.Printf("Name: %s\nContents: %v\nIndividual Value: %s\nBundle Price: %s\nDiscount: %s\nOne-Time Purchase: %t\nExpected Conversion: %s\n",
		pack.Name, pack.Contents, pack.IndividualValue, pack.BundlePrice, pack.Discount, pack.OneTimePurchase, pack.ExpectedConversion)
}
